free-1.md

```diff
@@ -10,34 +10,48 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+My idea would help physics students (or just people in general) do simulations involving some basic forces (like gravity and magnetic forces). As of right now, many physics students don't have very good access to simple simulators that they can play around with. There are some online, but they're written in Java and are very difficult to get working. This DSL would hopefully make it easy to set up simple demos and create animations of the results of the simulaiton.
```

Comment:
So the target is specifically students, rather than professionals? I think that's a fine choice but just make sure that you're focused on that choice in motivating design decisions. This is very similar to my first idea in a sense, although my hope was for mine to be applicable to both, and thinking about it it may be better to focus on one.